plenary/plenary-31.yaml

```diff
@@ -0,0 +1,287 @@
+    identifier: 007-2012
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2012-10-12
+        message: with regret accepts the resignation of its WG 4 convenor Mr. Michael Dill, effective immediately. The TC expresses its appreciation to Mr. Michael Dill for his outstanding contribution to the work of ISO/TC 154. The Technical Committee wishes Mr. Michael Dill every success for the future.
+
+
+  - dates:
+      - 2012-10-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Appointment of Ms. Sue Probert as Convenor of ISO/TC 154/WG 4"
+    identifier: 008-2012
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2012-10-12
+        message: resolves to appoint Ms. Sue Probert as Convenor of Working Group 4 eLK, effective immediately
+      - type: requests
+        date_effective: 2012-10-12
+        message: requests DIN to continue to offer the secretariat of ISO/TC 154/WG 4
+
+
+  - dates:
+      - 2012-10-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "ISO/PWI TR 18262 to DTR ballot"
+    identifier: 009-2012
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: resolves
+        date_effective: 2012-10-12
+        message: resolves to submit ISO/PWI TR 18262 Open Data Interchange Framework (ODIF) to DTR ballot, and submit the Draft to the liaison organization for comments
+
+
+  - dates:
+      - 2012-10-12
+    subject: ISO/TC 154 Processes, data elements and documents in commerce, industry and administration
+    title: "Reinstatement all withdrawn versions of ISO 9735"
+    identifier: 010-2012
+    considerations:
+      - type: considering
+        date_effective: 2012-10-12
+        message: noting that older versions of the EDIFACT syntax (ISO 9735) were recently withdrawn in the ISO database
+      - type: considering
+        date_effective: 2012-10-12
+        message: reminding that there was an older TC 154 resolution on retaining older EDIFACT syntax versions which is still valid
+      - type: considering
+        date_effective: 2012-10-12
+        message: recognizing that there is still a user/business need to keep the older EDIFACT syntax versions available
+
+    approvals:
+      - type: affirmative
+        degree: unanimous
+        message: The resolution was taken by unanimity
+
+    actions:
+      - type: requests
+        date_effective: 2012-10-12
+        message: requests ISO/TMB to reinstate all withdrawn versions of ISO 9735
```

Comment:
In this case (and some others), I took the list items for considerations (see original source). Is it correct?

Original source:
![capture8](https://user-images.githubusercontent.com/33627611/71759361-6991a400-2e82-11ea-9ddc-311c69073870.JPG)

Reply:
Good point. These should be keyed as "noting", "reminding" and "recognising".